marklogic-data-hub-central/ui-custom/src/components/ImageGallery/ImageGallery.test.tsx

```diff
@@ -0,0 +1,84 @@
+
+import {render, waitFor} from "@testing-library/react";
+import {DetailContext} from "../../store/DetailContext";
+import ImageGallery from "./ImageGallery";
+
+const configMultiple = {
+  component: "ImageGallery",
+  config: {
+    style: {
+      height: "150px",
+      width: "150px"
+    },
+    images: {
+      url: {
+        arrayPath: "person.images.image",
+        path: "url"
+      }
+    },
+    metadata: {},
+    download: true
+  }
+};
+
+const detail = {
+  person: {
+    images: {
+      "image": [
+        {
+          "url": "https://cdn1.marklogic.com/wp-content/uploads/2018/01/Diane-Burley-MarkLogic-2015.jpg",
+          "source": {
+            "name": "Washington Post",
+            "ts": "2011-09-29T17:38:02Z"
+          }
+        },
+        {
+          "url": "https://cdn1.marklogic.com/wp-content/uploads/2018/01/Alicia-Saia-MarkLogic1.jpg",
+          "source": {
+            "name": "Wall Street Journal",
+            "ts": "2012-04-10T16:00:11Z"
+          }
+        },
+        {
+          "url": "https://cdn1.marklogic.com/wp-content/uploads/2021/02/1612313387205.jpeg",
+          "source": {
+            "name": "Los Angeles Times",
+            "ts": "2013-10-28T05:20:37Z"
+          }
+        },
+        {
+          "url": "https://cdn1.marklogic.com/wp-content/uploads/2018/02/trinh-lieu-profile.jpg",
+          "source": {
+            "name": "New York Times",
+            "ts": "2012-11-12T22:50:47Z"
+          }
+        }
+      ]
+    }
+  }
+};
+
+const detailContextValue = {
+  detail: detail,
+  handleDetail: jest.fn()
+};
+describe("ImageGallery component", () => {
+
+  test("Verify ImageGallery widget renders correctly", () => {
+    const {getByTestId} = render(
+      <DetailContext.Provider value={detailContextValue}>
+        <ImageGallery config={configMultiple.config} data={detail} />
```

Comment:
@iveyMarklogic, it looks like these two tests are nearly identical, same inputs? Can they be combined into one test?